src/exact_cover.c

```diff
@@ -39,13 +39,13 @@ static PyObject* get_exact_cover(PyObject* self, PyObject* args)
     /*  Parse single numpy array argument */
```

Comment:
just seen l37:
`int *in_array_data, rows, cols, result;`
which also needs to be fixed, sorry!

Reply:
Fixed that too!